Fri May 28 10:40:32 CEST 2010  Lennart Kolmodin <kolmodin@gentoo.org>
  * Point out where to find header files in .cabal
  This makes haddock build the docs.
diff -rN -u old-ghc-events/ghc-events.cabal new-ghc-events/ghc-events.cabal
--- old-ghc-events/ghc-events.cabal	2010-05-28 10:43:19.329416454 +0200
+++ new-ghc-events/ghc-events.cabal	2010-05-28 10:43:19.331415871 +0200
@@ -21,6 +21,7 @@
                     containers >= 0.2 && < 0.4,
                     binary     == 0.5.*,
                     bytestring == 0.9.*
+  include-dirs:     GHC/RTS
   exposed-modules:  GHC.RTS.Events
 }
 
